#Definition of the defuzzyfied output type
#the name of the input variable
string name
#the defuzzyfied value
float64 value